### PR TITLE
Add docstring to skimage.draw module

### PR DESCRIPTION
**Pull request by anand-pb**
## Description

Added docstrings to skimage.draw .This updates the issue https://github.com/scikit-image/scikit-image/issues/6761

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

File: skimage/draw/__init__.py
Comment (lagru) [inline skimage/draw/__init__.py:6]: Hey @anand-pb, thanks for the first contribution! It's a good start, however, there are some formatting and style issues to improve upon. Please have a look at the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html) which also describes docstring conventions that skimage is using. 

It's a good idea to look for [examples in our library itself](https://github.com/scikit-image/scikit-image/blob/44da98e0a61f9382b0d96ad3973a1b83b2db1b5f/skimage/io/__init__.py#L1-L5).

What do you think about shortening this description to a single line?

Comment (anand-pb) [reply skimage/draw/__init__.py:6]: yes @lagru, possible.
should I be making an other pull request ?

Comment (lagru) [reply skimage/draw/__init__.py:6]: There is no need. :D You should be able to continue updating this pull request by pushing to your branch!

Comment (anand-pb) [reply skimage/draw/__init__.py]: ok, I have pushed the changes now, checks are going on

Comment (anand-pb) [reply skimage/draw/__init__.py]: it passed all the checks, can it be merged ?

**Comment by anand-pb** [comment]
> Dear @anand-pb,
> 
> Thank you for your first contribution! A good practice is a to write something in the description field of the PR. In this case, I agree that there isn't much to say; typically, you could refer to the issue you're addressing (here, #6761).

I edited the description of the pull request, will this be sufficient ?

**Review by lagru — APPROVED** [review]

File: skimage/draw/__init__.py
Comment (lagru) [reply skimage/draw/__init__.py]: Once a core developer approves, in this case, yes it can be merged. If you are interested in how and when changes are merged, have a look at our [decision making guidelines](https://scikit-image.org/docs/dev/skips/1-governance.html#decision-making-process). 

I've left another suggestion you are free to ignore. After you've commented or committed the suggestion will merge this. ;)

File: skimage/draw/__init__.py
Comment (anand-pb) [reply skimage/draw/__init__.py:2]: so, where do you want this long description to be written ?

Comment (mkcor) [reply skimage/draw/__init__.py]: > Looks great! What do you think about using "functionality" instead of "utilities". Does that sound better to you?

@lagru I don't agree... I think that 'utilities' could perhaps be replaced by 'utility functions' or 'helper functions' (but 'utilities' is fine). 'Functionality' has a slightly different meaning, it's more abstract; the _functionality_ is what's provided by the helper function, it's the 'result' of using it... I think that "functionality for generating coordinates (...)" doesn't really work, because the generation of coordinates is the functionality itself.

Comment (anand-pb) [reply skimage/draw/__init__.py]: should I do something now ?

Comment (lagru) [reply skimage/draw/__init__.py]: I find @mkcor arguments, as usually, very convincing. I think I mainly suggested it because our submodules are all starting to look like "Utilities for ..." which I guess prompted me to want to mix it up. :smile: Just leave it like it is if you agree with @mkcor.

> so, where do you want this long description to be written ?

Just below the first line (leaving a blank line between). In https://github.com/scikit-image/scikit-image/issues/6761 I linked a style guide to writing docstrings. It is the same pattern that is used in a lot of scikit-images functions: Short description in imperative mood, followed by a more detailed description of the purpose and use.

Comment (anand-pb) [reply skimage/draw/__init__.py]: I referred numpy github repository files for detailed docstrings, let me know if this will do @lagru

Comment (anand-pb) [reply skimage/draw/__init__.py]: @lagru  detailed description docstring has been added to skimage.draw module, and the corresponding pull request is at #6804

Comment (anand-pb) [reply skimage/draw/__init__.py]: yes, #6804 supersedes this PR, so I am going to close this PR

**Pull request closed without merging**
